.github/workflows/lint.yml

```diff
@@ -7,11 +7,18 @@ jobs:
     name: Check Spelling
     runs-on: ubuntu-latest
     steps:
-      - name: Check Out
+      - name: Check out
         uses: actions/checkout@v2
       - name: Install
         run: |
           wget -O - -q https://git.io/misspell | sh -s -- -b .
       - name: Misspell
         run: |
           git ls-files --empty-directory | xargs ./misspell -error
+  trailing-whitespace:
+    name: Trailing Whitespace
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check for trailing whitespace
```

Comment:
Would it make sense to add it as a step on the other one to have a single linting job?